modules/richards/tests/dirac/tests

```diff
@@ -78,6 +78,7 @@
     input = 'bh27.i'
     exodiff = 'bh27.e'
     rel_err = 1E-5
+    abs_zero = 1E-5
```

Comment:
yea, the units are in 1E6 units, so abs_zero=1E-5 is still quite small.  (Also, this is a Richards test that is not incredibly important.)